### ISSUE TITLE
fillscreen has no effect?

### ISSUE THREAD

**Issue by zach-sb** [damongolding/immich-kiosk#20]
When trying to set `fill_screen=true` it has no obvious effect. However, if I edit the css for fillscreen from

```css
.frame--image-fillscreen {
    width: 100%;
    height: 100%;

    object-fit: contain;
}
```
![Screenshot 2024-07-27 085141](https://github.com/user-attachments/assets/139f2d49-494e-4a0e-9a10-0439d4477984)
to 
```css
.frame--image-fillscreen {
    width: 100%;
    height: 100%;

    object-fit: cover;
}
```

![Screenshot 2024-07-27 085121](https://github.com/user-attachments/assets/14ed9b05-ff96-437c-b5f1-f2162d8ddd79)

The image displays as I would expect from the name. I'm okay with it being centered like it does by default, I just didn't want any background at all.

**Comment by damongolding** [comment]
Ahh I can see where the confusion has come from. What fill_screen currently does is upscale the image to fill the viewport without cropping any of the image. 

As an example. If you had a screen size of 1920x1080 but the image is only 1280x720, fill_screen would scale up the image to be 1920x1080.

**Comment by zach-sb** [comment]
Ah, that makes sense why it didn't seem to be doing anything then, nearly all of my photos are larger than my screens' resolutions!

In that case can I request an option for having a cover mode (`<url>?cover=true`)? Also, wouldn't `upscale` be a better name than `fill_screen` then?

**Comment by zach-sb** [comment]
Is there actually anything other than css object fitting going on there? Because if not perhaps a `<url>?image-fit=[cover|contain|fill|etc]` would be nice?

**Comment by damongolding** [comment]
> Is there actually anything other than css object fitting going on there? Because if not perhaps a `<url>?image-fit=[cover|contain|fill|etc]` would be nice?

That's an excellent suggestion!

I was also thinking of utilising object-position as Immich can supply face positions. Then Kiosk could try and centre/avoid cropping out those. But centring the image would be a quick win.

**Comment by damongolding** [comment]
I have implemented this in 0.4.0.

Again, thanks for the suggestion!
